astropy/cosmology/units.py

```diff
@@ -54,18 +66,22 @@
 # Equivalencies
 
 
-def dimensionless_redshift():
+def dimensionless_redshift() -> Equivalency:
     """Allow redshift to be 1-to-1 equivalent to dimensionless.
 
     It is special compared to other equivalency pairs in that it allows
     this independent of the power to which the redshift is raised, and
     independent of whether it is part of a more complicated unit. It is
     similar to u.dimensionless_angles() in this respect.
     """
-    return u.Equivalency([(redshift, None)], "dimensionless_redshift")
+    return Equivalency([(redshift, None)], "dimensionless_redshift")
 
 
-def redshift_distance(cosmology=None, kind="comoving", **atzkw):
+def redshift_distance(
+    cosmology: Cosmology | str | None = None,
+    kind: Literal["comoving", "lookback", "luminosity"] = "comoving",
```

Comment:
tangent, but I notice that the docstring doesn't say which option is the default, maybe now is as good a time as any to fix this.

Reply:
The first value listed in braces is the default and it does not need to be highlighted more than that: https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
Today I learned. Thanks !